File: app/LIFE AFTER GRADUATED/app/src/main/java/com/example/anemoneapp/data/model/Teacher.kt
package com.example.anemoneapp.data.model

data class Teacher(
    val id: Int,
    val nip: String,
    val full_name: String,
    val subject_id: Int,
    val phone_number: String,
    val status: String,
    val created_at: String,
    val updated_at: String
)

data class TeacherRequest(
    val nip: String,
    val full_name: String,
    val subject_id: Int,
    val phone_number: String,
    val status: String
)

data class TeacherResponse(
    val id: Int,
    val nip: String,
    val full_name: String,
    val subject_id: Int,
    val phone_number: String,
    val status: String,
    val created_at: String,
    val updated_at: String
)

File: app/LIFE AFTER GRADUATED/test/school-management/app/Models/Teacher.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;

class Teacher extends Model
{
    use HasFactory;

    protected $fillable = [
        'nip',
        'full_name',
        'subject_id', 
        'phone_number',
        'status',
    ];

    public function subject()
    {
        return $this->belongsTo(Subject::class);
    }

    public function schedules()
    {
    return $this->hasMany(Schedule::class);
    }
}
